scripts/check_tabs.sh

```diff
@@ -30,6 +30,9 @@ declare -a excluded_files_patterns=(
   "\.jar$"
   "\.so$"
   "\.vmfb$"
+  "\.jpg$"
+  "\.png$"
+  "\.svg$"
```

Comment:
Ok people keep adding excludes instead of taking my suggestion to add the grep flag that ignores binary files... I guess I'll have to do that myself :stuck_out_tongue:

Reply:
Please do :P

Reply:
https://github.com/google/iree/pull/5739

Reply:
Thanks :) rebased on top of that.